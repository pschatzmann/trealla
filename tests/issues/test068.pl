:- initialization(main).

main :-
    prepare(List),
    writeln(List),
    sort(List, ListSorted),
    writeln(ListSorted).

prepare([B,A]) :-
    A =.. [pair,2,X],
    B =.. [trio,3,Y,Z].
