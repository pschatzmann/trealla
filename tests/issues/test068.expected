[trio(3,_6,_7),pair(2,_5)]
[pair(2,_5),trio(3,_6,_7)]
